UCLH/analysis/most_common_bundles.csv

```diff
@@ -0,0 +1,299 @@
+BundleName,count
```

Comment:
Just to be consistent with the rest of the package, could we use `bundle_name` instead of `BundleName`?